src/edu.umn.cs.melt.copper.compiletime/src/main/java/edu/umn/cs/melt/copper/compiletime/spec/numeric/ParserSpec.java

```diff
@@ -183,17 +184,22 @@ public ProductionData(int count,int maxRHS)
 		protected BitSet[] members;
 		/** The terminal to which the disambiguation group disambiguates, or -1 for disambiguation functions that disambiguate using code. */
 		protected int[] disambiguateTos; // -1 for all but disambiguation groups.
+		/** If true, this disambiguation function should be applied for any subset of members. */
+		protected boolean[] applicableToSubsets;
```

Comment:
If this were implemented as a single `BitSet` rather than a boolean array, it would be possible to put all the conditionals in order in the engine builder without iterating through all disambiguation functions twice.